InnoClinic.AuthorizationAPI/API/Controllers/AuthController.cs

```diff
@@ -0,0 +1,31 @@
+﻿using BLL.DTO;
+using BLL.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class AuthController(IAuthService authService) : ControllerBase
+{
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [HttpPost("register")]
+    public async Task<IActionResult> RegisterAsync([FromBody] RegisterDto dto, CancellationToken cancellationToken)
+    {
+        var result = await authService.RegisterAsync(dto, cancellationToken);
+        if (!result.Succeeded) return BadRequest(result.Errors);
+        return Ok(result);
+    }
+
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [HttpPost("login")]
+    public async Task<IActionResult> LogInAsynс([FromBody] LogInDto dto, CancellationToken cancellationToken)
+    {
+        var result = await authService.LogInAsync(dto, cancellationToken);
+        if (!result.Succeeded) return Unauthorized(new {Message = Messages.UserIsNotLoggedIn});
```

Comment:
you don't need to create new object, just provide a string inside Unauthorized()